ddp-workspace/projects/ddp-dsm-ui/src/app/sequencing-order/sequencing-order.component.ts

```diff
@@ -0,0 +1,162 @@
+import {Component, Input, ViewChild} from '@angular/core';
+import {ModalComponent} from '../modal/modal.component';
+import {Participant} from '../participant-list/participant-list.model';
+import {Auth} from '../services/auth.service';
+import {ComponentService} from '../services/component.service';
+import {DSMService} from '../services/dsm.service';
+import {RoleService} from '../services/role.service';
+import {PatchUtil} from '../utils/patch.model';
+import {Utils} from '../utils/utils';
+import {SequencingOrder} from './sequencing-order.model';
+
+@Component( {
+  selector: 'app-sequencing-order',
+  templateUrl: './sequencing-order.component.html',
+  styleUrls: [ './sequencing-order.component.scss' ]
+} )
+export class SequencingOrderComponent {
+  @Input() participant: Participant;
+  @Input() samples: SequencingOrder[];
+
+  @ViewChild( ModalComponent )
+  public modal: ModalComponent;
+
+  errorModal = false;
+
+  currentPatchField: string;
+  errorMessage: string;
+  selectedNormal: SequencingOrder = new SequencingOrder( '', '', '', '', '', '', '', '' );
+  selectedTissue: SequencingOrder = new SequencingOrder( '', '', '', '', '', '', '', '' );
```

Comment:
Here I would make properties optional or just set default values in the class constructor.

Reply:
Here I just need an empty instance to set it to what they select. I don't want it to be default values, because I want us to know if something is missing  for the Order class, and I don't want them to be optional. So I changed it to just create Empty instances. What do you think about the new changes?